api/src/services/collections.ts

```diff
@@ -377,7 +377,9 @@ export class CollectionsService {
 		}
 
 		await this.knex.transaction(async (trx) => {
-			await trx.schema.dropTable(collectionKey);
+			if (collectionToBeDeleted!.schema !== null) {
+				await trx.schema.dropTable(collectionKey);
+			}
```

Comment:
Lets use the same check as below on 397

```suggestion
			if (collectionToBeDeleted!.schema) {
				await trx.schema.dropTable(collectionKey);
			}
```